Python/compile.c

```diff
@@ -911,7 +911,14 @@ PyObject *
 _PyCompile_StaticAttributesAsTuple(compiler *c)
 {
     assert(c->u->u_static_attributes);
-    return PySequence_Tuple(c->u->u_static_attributes);
+    PyObject *static_attributes_unsorted = PySequence_List(c->u->u_static_attributes);
+    if (static_attributes_unsorted == NULL) {
+        return NULL;
+    }
+    PyList_Sort(static_attributes_unsorted);
```

Comment:
```suggestion
    if (PyList_Sort(static_attributes_unsorted) < 0) {
        Py_DECREF(static_attributes_unsorted);
        return NULL
    }
```

Reply:
fixed (to `!=0` to be consistent with [other code in this file](https://github.com/python/cpython/blob/e097874047a830f087022bf210761b9b595c72c6/Python/compile.c#L515))